app/views/external_users/claims/basic_fees/_additional_fee_fields.html.haml

```diff
@@ -13,12 +13,20 @@
     = f.hidden_field :fee_type_id, class: 'js-fee-type'
 
     - if f.object.fee_type.case_uplift?
-      .form-group
-        = f.adp_text_field :case_numbers,
-          label: t('.case_numbers'),
-          input_classes: 'js-basic-fee-case-numbers fx-fee-case-numbers',
-          hint_text: 'Separate by commas',
-          errors: @error_presenter
+      - if Settings.urn_enabled?
```

Comment:
use dynamic translation instead
```
option 1:
label: t('.case_numbers', urn) : t('.case_numbers', addendum: Settings.urn_enabled? ? 'or URN' : '')

option 2
label: Settings.urn_enabled? ? t('.case_numbers_or_urn') : t('.case_numbers')

option 3
label: Settings.urn_enabled? ? t('.case_numbers.with_urn') : t('.case_numbers.without_urn')

```

Reply:
👍 thank you for the suggestion, I have amended to use dynamic translations (option 2).